erdjs/src/smartcontracts/codec/numerical.ts

```diff
@@ -83,26 +73,72 @@ export class NumericalBinaryCodec {
         return Buffer.concat([length, buffer]);
     }
 
+    private encodeNestedFixedSize(primitive: NumericalValue, size: number): Buffer {
```

Comment:
what if the primitive's size is larger than the provided size? Should we have an `if` for this or we rely on the caller to call the function with correct values?

Reply:
It will never be, It's a question I also had and discussed it with @andreibancioiu, and decided that it's safe to leave it like this